src/constraint_aware_append.c

```diff
@@ -567,10 +567,6 @@ ts_constraint_aware_append_possible(Path *path)
 	ListCell *lc;
 	int num_children;
 
-	if (ts_guc_disable_optimizations || !ts_guc_constraint_aware_append ||
```

Comment:
why is this removed?

Reply:
Good question. Might be a cherry-pick/rebase artifact. I'll double check.

Reply:
Added back. Still, some of these checks are unnecessary since some GUCs are checked multiple times at this point. Might want to overhaul this at some point.